app/services/doc_auth/mock/doc_auth_mock_client.rb

```diff
@@ -75,10 +75,13 @@ def post_images(
         back_image_response = post_back_image(image: back_image, instance_id: instance_id)
         return back_image_response unless back_image_response.success?
 
-        get_results(instance_id: instance_id)
+        get_results(
+          instance_id: instance_id,
+          selfie_required: !selfie_image.blank? || liveness_checking_required,
```

Comment:
I think it's clearer to use one of these or the other.

I have a slight preference for this: ` selfie_required: liveness_checking_required,`

Reply:
@charleyf , just is thinking about the same and I think I agree.

Reply:
Looks good! Did a quick test and the image upload still works.